src/model/Home.js

```diff
@@ -1,17 +1,7 @@
 const mongoose = require('mongoose');
 
-const HomePageSchema = mongoose.Schema({
-	title: String,
-	priority: Number,
-	type: String,
-	data: [
-		{
-			url: String,
-			title: String,
-			imageUrl: String,
-			description: String,
-		},
-	],
-});
+// we want home API to be flexible 
+// not validating any data type or model defination
 
-module.exports = mongoose.model('home', HomePageSchema);
+const HomePageSchema = new mongoose.Schema({}, { strict: false });
```

Comment:
if it is not strict then we shall not need schema then anyway user will be able to make new field entry if field type is different also

Reply:
It's just hack to create record in mongo using mongoose without any schema